stylesheets/colours/_organisation.scss

```diff
@@ -18,6 +18,8 @@ $department-for-education-websafe: #347ca9;
 $department-for-environment-food-rural-affairs: #00a33b;
 $department-for-international-development: #002878;
 $department-for-international-development-websafe: #405e9a;
+$department-for-international-trade: #CF102D;
+$department-for-international-trade-websafe: $link-colour;
```

Comment:
This mirrors UKTI https://github.com/alphagov/govuk_frontend_toolkit/blob/9bb073ca043433e4577935100dedc8998bc4479e/stylesheets/colours/_organisation.scss#L59

We could have all the links in pink but It didn't seem appropriate will get an opinion from design.